setup.py

```diff
@@ -17,16 +17,15 @@
 
 from setuptools import setup, find_packages
 
-
-# Hack required to allow compat to not fail when pypiwin32 isn't found
-os.environ["PYINSTALLER_NO_PYWIN32_FAILURE"] = "1"
-
-
 #-- plug-in building the bootloader
 
 from distutils.core import Command
 from distutils.command.build import build
 
+
+# Hack required to allow compat to not fail when pypiwin32 isn't found
+os.environ["PYINSTALLER_NO_PYWIN32_FAILURE"] = "1"
+
```

Comment:
Was this move automatic? I'd be quite worried if it was. This one looks harmless but normally there is a reason for not putting imports at the top of files.

Reply:
No, move was manual :P I got a warning from flake8 so I decided to fix it.